src/globus_cli/commands/api.py

```diff
@@ -240,6 +254,16 @@ def build_command(service_name: ServiceNameLiteral) -> click.Command:
         ),
     )
     @click.option("--no-retry", is_flag=True, help="Disable built-in request retries")
+    @click.option(
+        "--scope-string",
+        type=str,
+        multiple=True,
+        help=(
+            "A scope string to consent to for this command (only supported for "
+            "confidential client usage). "
+            "Pass this option multiple times to specify multiple scopes."
```

Comment:
Recommending removal of "for this command". I've rewrapped the lines as well.

```suggestion
            "A scope string to consent to "
            "(only supported for confidential client usage). "
            "Pass this option multiple times to specify multiple scopes."
```

Reply:
Only about readability -- "to consent to for" caught my eye.